src/main.ts

```diff
@@ -31,23 +20,40 @@ async function mapiCLI(): Promise<string> {
     os = 'linux-musl'
   }
 
+  // Get latest version from API
+  let cliVersion = 'latest'
+  try {
+    cliVersion = (await cliInfo(os, bin)).version
+  } catch (err: unknown) {
+    if (err instanceof Error) {
+      core.info(err.message)
+      core.debug('Could not get CLI version. Using latest')
+    }
+  }
+
   // Return cache if available
-  const cachedPath = tc.find('mapi', cliVersion, os)
+  const cachedPath = tc.find('security.mayhem.app.cli.mapi', cliVersion, os)
```

Comment:
I think we need an if guard around `cachedPath` logic if `cliVersion === 'latest'` -- we don't ever want to fetch `'latest'` from the cache.